src/app/app.component.html

```diff
@@ -104,7 +104,7 @@ <h1 class="color1">Recruitment Status</h1>
             Matching Trials</span>
           <div>
             <div *ngFor="let filter of filtersArray;let i = index;" class="filters mi-margin-10"
-              style=" position: relative;display: inline-block;"> <span class="color1 filterText">
+              style=" position: relative;display: inline-block;"> <span class="color1 filterText pointer" :hover>
```

Comment:
What is `:hover`? `:hover` is a CSS "pseudoclass" - it represents the element when it's being hovered, it doesn't need to be in the HTML. I think syntactically `:hover` is valid as a namespace-less attribute called `hover` but it doesn't need to be in the HTML.